src/main/resources/assets/bloodmagic/lang/de_DE.lang

```diff
@@ -1,181 +1,256 @@
+tile.BloodMagic.ritualStone.dusk.name=Morgendämmerungsritualstein
+tile.BloodMagic.ritualStone.dawn.name=Abenddämmerungsritualstein
+
+tile.BloodMagic.bloodstonebrick.normal.name=Blutsteinziegel
+tile.BloodMagic.bloodstonebrick.large.name=Großer Blutsteinziegel
+tile.BloodMagic.crystal.normal.name=Kristallklumpen
+tile.BloodMagic.crystal.brick.name=Kristallklumpenziegel
+tile.BloodMagic.bloodLight.name=Blutlampe
+tile.BloodMagic.spectralBlock.name=Spektralblock
+tile.BloodMagic.phantomBlock.name=Phantomblock
+
+tile.BloodMagic.teleposer.name=Teleposer
+
+# Tooltips
+tooltip.BloodMagic.orb.desc=Speichert Lebensessenz
+tooltip.BloodMagic.orb.owner=Hinzugefügt durch: %s
+tooltip.BloodMagic.currentOwner=Eigentümer: %s
+tooltip.BloodMagic.currentTier=Ausbaustufe: %d
+
+tooltip.BloodMagic.activated=Aktiviert
+tooltip.BloodMagic.deactivated=Deaktiviert
+
+tooltip.BloodMagic.sigil.air.desc=&oIch fühl mich schon leichter...
+tooltip.BloodMagic.sigil.bloodLight.desc=&oIch sehe ein Licht!
+tooltip.BloodMagic.sigil.compression.desc=&oDiamanthände
+tooltip.BloodMagic.sigil.divination.desc=&oSpähe in die Seele
+tooltip.BloodMagic.sigil.divination.currentAltarTier=Ausbaustufe: %d
+tooltip.BloodMagic.sigil.divination.currentEssence=Gespicherte Lebensessenz: %d LP
+tooltip.BloodMagic.sigil.divination.currentAltarCapacity=Maximaler Kapazität: %d LP
+tooltip.BloodMagic.sigil.water.desc=&oHat jemand nach unendlichem Wasser gefragt?
+tooltip.BloodMagic.sigil.lava.desc=&oHEIẞ! NICHT ESSEN!
+tooltip.BloodMagic.sigil.void.desc=&oBesser als Swiffer®!
+tooltip.BloodMagic.sigil.greenGrove.desc=&oUmweltverträglich
+tooltip.BloodMagic.sigil.magnetism.desc=&oIch habe eine anziehende Persönlichkeit
+tooltip.BloodMagic.sigil.suppression.desc=&oBesser als Telekinese
+tooltip.BloodMagic.sigil.haste.desc=&o42 Dosen Koffein später...
+tooltip.BloodMagic.sigil.fastMiner.desc=&oHacke wei ein Weltmeister
+tooltip.BloodMagic.sigil.elementalAffinity.desc=&oAchtung! Fallende Feuerfische
+tooltip.BloodMagic.sigil.seer.desc=&oWenn allsehend nicht genug ist
+tooltip.BloodMagic.sigil.seer.currentAltarProgress=Fortschritt: %d LP/ %s LP
+tooltip.BloodMagic.sigil.seer.currentAltarProgress.percent=Fortschritt: %s
+tooltip.BloodMagic.sigil.seer.currentAltarConsumptionRate=Verbrauch: %d LP
+tooltip.BloodMagic.sigil.seer.currentAltarTier=Ausbaustufe: %d
+tooltip.BloodMagic.sigil.seer.currentEssence=Gespeicherte Lebensessenz: %d LP
+tooltip.BloodMagic.sigil.seer.currentAltarCapacity=Maximaler Kapazität: %d LP
+tooltip.BloodMagic.sigil.seer.currentCharge=Ladung: %d
+tooltip.BloodMagic.sigil.phantomBridge.desc=&oLaufen auf dünner Luft...
+tooltip.BloodMagic.sigil.whirlwind.desc=&oBesser keinen Rock tragen
+tooltip.BloodMagic.sigil.enderSeverance.desc=&oBringt Enderman in Notlage
+
+tooltip.BloodMagic.bound.sword.desc=&oErlegen der Schwachen
+tooltip.BloodMagic.bound.pickaxe.desc=&oZerstören von Stein ohne Erbarmen
+tooltip.BloodMagic.bound.axe.desc=&oDemonische Entwälderung
```

Comment:
"Dämonische Entwaldung" would be the correct translation. Personally, I'd go with "Dämonsicher Kahlschlag" which, IMO, sounds less dry.